README.md

```diff
@@ -107,41 +107,47 @@ packager can transfer the source code over USB instead of via Wi-Fi, especially
 in networks that block your computer from connecting to your phone, like in
 public places.
 
-#### ... on iOS:
+## Troubleshooting
 
-1.  Open `ios/Raha.xcworkspace` (not `Raha.xcodeproj`!) in XCode, or by
```

Comment:
I found these instructions useful in the past.. are they no longer necessary?

Reply:
Ah word I think I meant to move them elsewhere but ended up just deleting them, bring it back